src/psyclone/gocean1p0.py

```diff
@@ -487,8 +461,9 @@ def __init__(self, parent=None,
         :param str topology_name: Optional opology of the loop (unused atm).
         :param str loop_type: Loop type - must be 'inner' or 'outer'.'''
         # pylint: disable=unused-argument
+        const = GOceanConstants()
```

Comment:
A small thing but could you move the arg-description part of the docstring above into that for the class as a whole and then delete the remainer.

Reply:
Done.